hledger-lib/Hledger/Read/Common.hs

```diff
@@ -830,7 +831,7 @@ numberp suggestedStyle = label "number" $ do
       Left errMsg -> Fail.fail errMsg
       Right (q, p, d, g) -> pure (sign q, p, d, g)
 
-exponentp :: TextParser m Int
+exponentp :: TextParser m Integer
```

Comment:
let's mention this in the commit message too. If we get around to it we might even come up with an example showing the improvement, for release notes

Reply:
Mentioning this you mean the use of `Word8` in the type of `numberp`? (I feel like the comments are offset from what you're referring to)